lib/spacesuit/cors_middleware.ex

```diff
@@ -0,0 +1,166 @@
+defmodule Spacesuit.CorsMiddleware do
+  require Logger
+  use Elixometer
+
+  @http_server            Application.get_env(:spacesuit, :http_server)
```

Comment:
This thing needs to stand in for the calls to `:cowboy_req` for purposes of mocking this out in tests.